diagnostic.md

```diff
@@ -71,13 +71,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative.  It doesn't contain the full address.  You could not get to the correct location unless you started from the project folder.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Relative.  If you ever move these files to a different location, the image will still show up with a relative file path.  But if it's absolute, it will break.
```

Comment:
It is relative, but because the image file lives inside our current working directory, project ; as a result, an absolute path is not needed to access it.